object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -611,6 +612,7 @@ private void handleSuccessorBlocks(
           Set<LocalVarWithTree> setOfLocals = new LinkedHashSet<>();
           setOfLocals.add(new LocalVarWithTree(new LocalVariable(paramElement), param));
           init.add(ImmutableSet.copyOf(setOfLocals));
+          increaseNumMustCall();
```

Comment:
Why do we also increment here?  Please add a comment

Reply:
https://github.com/kelloggm/object-construction-checker/pull/249/commits/5260a7c49c6dc2c042e59816683cc968c422883c